src/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/Components/UploadPhotosForm.tsx

```diff
@@ -52,7 +52,9 @@ export const UploadPhotosForm: React.FC<UploadPhotosFormProps> = ({
       photo.loading = false
 
       if (!geminiToken) {
-        photo.errorMessage = `Photo could not be added: ${photo.name}`
+        photo.errorMessage = photo.externalUrl
+          ? "Artwork image could not be automatically added. Please add a photo."
+          : `Photo could not be added: ${photo.name}`
```

Comment:
**thought:** not sure if the name here is going to be descriptive enough since it's probably going to be something random

Reply:
Good point! Let's tackle this separately to see if we can make the copy here more clear as well.